src/TextField/TextField.js

```diff
@@ -47,6 +47,7 @@ const getStyles = (props, context, state) => {
     },
     floatingLabel: {
       color: hintColor,
+      pointerEvents: 'none',
```

Comment:
I'm not sure about this fix:
![capture d ecran 2016-06-09 a 23 08 50](https://cloud.githubusercontent.com/assets/3165635/15946417/2256ed78-2e97-11e6-8acb-54bd11dd31c8.png)